### PR TITLE
Helper blueprints for Module Unification

### PR DESCRIPTION
**Pull request by ppcano**
Closes #16492 and #16944

Listed on the MU Quest issue ember-cli/ember-cli#7530

### CODE REVIEW

File: node-tests/blueprints/helper-test-test.js [hunk 1]
```diff
@@ -137,6 +138,130 @@ describe('Blueprint: helper-test', function() {
...
+      it('helper-test foo/bar-baz for mocha --unit', function() {
+        return emberGenerateDestroy(['helper-test', 'foo/bar-baz', '--unit'], _file => {
+          expect(_file('src/ui/components/foo/bar-baz-test.js')).to.equal(
+            fixture('helper-test/mocha-0.12-unit.js')
+          );
+        });
+      });
+    });
+    /*
+    // TODO: this test is failing
```
[the remark this answers is not in the document]
Comment (simonihmig) [reply node-tests/blueprints/helper-test-test.js]: @ppcano Can't really say, without running the tests or seeing the test result. I *guess* the file contents are somewhat different than expected (in the fixture). Don't the tests show the diff?

**Review by NullVoxPopuli — APPROVED** [review]

File: node-tests/blueprints/helper-test-test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ppcano) [reply node-tests/blueprints/helper-test-test.js]: Fixed by https://github.com/emberjs/ember.js/pull/17211/commits/3a3b11687b508e251b4a0bcfa7721d2b150e307b

**Comment by ppcano** [comment]
@rwjblue This is ready for merging along with #17235.

I will have to refactor these MU blueprint tests for `ember-cli@3.5` at #17212

**Review by rwjblue — APPROVED** [review]
Thank you!

**Pull request merged**